### DISCUSSION TITLE
What happened with viem `1.4.0`?

### DISCUSSION THREAD

**Discussion by ennioVisco** [wevm/viem#948] [Question] [answered]
Hey there, 30% bundle size increase, but the changelog doesn't mention anything. 
It would be nice to know what changed!

<img width="133" alt="image" src="https://github.com/wagmi-dev/viem/assets/740025/c58bfe1e-dd6e-4bac-8d2f-0996ec5b8665">

**Comment by jxom** [comment, marked as answer]
Bundlephobia isn’t a realistic source for true bundle size (that is the total bundle size) - we will probably remove the link to bundlephobia shortly. Your application’s bundle size will be dependant on the modules you import (tree-shaking), so the size will be much lower than the one listed on Bundlephobia. 

Here is a more truthy representation: https://twitter.com/jakemoxey/status/1673846338067529730?s=46&t=_p0_kpxhTusXBkoJPXN4cg

**Comment by ennioVisco** [comment]
> Bundlephobia isn’t a realistic source for true bundle size (that is the total bundle size) - we will probably remove the link to bundlephobia shortly. Your application’s bundle size will be dependant on the modules you import (tree-shaking), so the size will be much lower than the one listed on Bundlephobia.
> 
> Here is a more truthy representation: https://twitter.com/jakemoxey/status/1673846338067529730?s=46&t=_p0_kpxhTusXBkoJPXN4cg

Thanks for the quick answer! I'm aware of it, but it still provides a good benchmark about the worst-case size. 
This increase probably means some major dependency has been included, it would be insightful to know what and why :)

**Reply by jxom** [reply]
It would have been Local Accounts (Private Key, BIP32, etc) introduced in 0.2 (where @noble/curves dependency was added).

For some reason, in your graphic, the last known npm version was 0.1.22. Unsure why Bundlephobia isn’t showing 0.2 onwards.

**Comment by ennioVisco** [comment]
Ah, I see, some versions are missing from bundlephobia, that's not unusual.

Thanks a lot!
